### PR TITLE
master-qa-41679

### PR DESCRIPTION
**Pull request by chaowang-123**
https://issues.liferay.com/browse/LRQA-41679
Submit to master.

### CODE REVIEW

**Comment by chaowang-123** [comment]
ci:test:sf

**Comment by chaowang-123** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase [hunk 1]
```diff
@@ -1189,6 +1189,85 @@ definition {
...
+			locator1 = "WC#ENTRY_LIST_TITLE_ON_LIVE",
+			value1 = "Web Content Title 2");
+
+		AssertElementPresent(
+			key_webContentTitle = "WC SubFolder Name",
+			locator1 = "WC#ENTRY_LIST_TITLE");
+
+		AssertElementNotPresent(
+			key_webContentTitle = "Web Content Title 1",
+			locator1 = "WC#ENTRY_LIST_TITLE_ON_LIVE");
```
Comment (brianwulbern) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase:1256]: @chaowang-123 I'll let this go this time, but as a best practice, we shouldn't have functions directly in the testcase file.  If we can wrap them into new macro commands and make them reusable that is best.

**Comment by brianwulbern** [comment]
No test changes needed :)

**Comment by brianwulbern** [comment]
Pushed to BChan where applicable on
master https://github.com/brianchandotcom/liferay-portal/pull/110040
7.3.x https://github.com/brianchandotcom/liferay-portal-ee/pull/35908
7.2.x https://github.com/brianchandotcom/liferay-portal-ee/pull/35909
7.1.x https://github.com/brianchandotcom/liferay-portal-ee/pull/35906
7.0.x https://github.com/brianchandotcom/liferay-portal-ee/pull/35907

**Pull request closed without merging**